.github/workflows/ci.yml

```diff
@@ -3,20 +3,25 @@ name: CI
 on: [push, pull_request]
 
 jobs:
-  build:
+  test:
+    name: test w/ Python
```

Comment:
Just tests should be enough. To show that it's not pypy we should say CPython, but pypy is likely to be in the matrix, so better to just name this tests